backend/api/viewsets/ComplianceReport.py

```diff
@@ -241,7 +300,7 @@ def get_latest_supplemental_reports(self):
         latest_supplementals = ComplianceReport.objects.raw("""
             select distinct on (p.id) c.*
             from compliance_report p 
-              left join compliance_report c on p.id = c.supplements_id
+            left join compliance_report c on p.id = c.id
```

Comment:
This seems odd. It's joining on itself and would only ever return one record. What was the intended purpose of this change?